interface/reports/rwt_2023_report.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/**
+ * Real world testing report for 2023.
+ *  Dates are hard-coded from 2023-04-01 to 2023-09-30
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Brady Miller <brady.g.miller@gmail.com>
+ * @copyright Copyright (c) 2022 Brady Miller <brady.g.miller@gmail.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+require_once("../globals.php");
+
+use OpenEMR\Common\Acl\AclMain;
+use OpenEMR\Common\Csrf\CsrfUtils;
+use OpenEMR\Common\Twig\TwigContainer;
+use OpenEMR\Core\Header;
+use OpenEMR\Reports\RealWorldTesting;
+
+if (!AclMain::aclCheckCore('admin', 'super')) {
+    echo (new TwigContainer(null, $GLOBALS['kernel']))->getTwig()->render('core/unauthorized.html.twig', ['pageTitle' => xl('2023 Real World Testing Report')]);
+    exit;
+}
+
+if (!empty($_POST)) {
+    if (!CsrfUtils::verifyCsrfToken($_POST['csrf_token_form'], 'rwt_2023_report')) {
+        CsrfUtils::csrfNotVerified();
+    }
+}
+
+// dates for this report are hard-coded (see header for details)
+$begin_date = '2023-04-01';
+$end_date = '2023-09-30';
+
+// can override dates below for development/testing purposes
+$begin_date = '2022-10-01';
+$end_date = '2022-12-01';
```

Comment:
Not I am overriding the dates here for now just for development/testing purposes. When this goes in the codebase, will instead use the official hard-coded dates of 2023-04-01 to 2023-09-30